modules/qtwidgets/src/tf/tfeditorprimitive.cpp

```diff
@@ -229,10 +231,11 @@ void TFEditorPrimitive::updateLabel() {
 
     QString label;
     if (auto tfe = qobject_cast<TFEditor*>(scene())) {
-        label = QString("%1 (%2) / %3")
-                    .arg(tfe->getDataMapper().mapFromNormalizedToValue(getPosition()))
-                    .arg(getPosition(), 0, 'g', 3)
-                    .arg(getColor().a, 0, 'g', 3);
+        label = utilqt::toQString(
+            fmt::format("{:0.3g} {} ({:0.3g}) / {:0.3g}",
+                        tfe->getDataMapper().mapFromNormalizedToValue(getPosition()),
+                        tfe->getDataMapper().valueAxis.unit, getPosition(), getColor().a));
```

Comment:
nitpicking: additional space if there is no unit.

Reply:
have a fix on the way